tgg/relatedness/doubleton_utils.py

```diff
@@ -31,14 +31,17 @@
 
 Generated using the following commands:
 
+# NOTE: If ever need to rerun, should read v4 vds using `get_gnomad_v4_vds`
+# (in gnomad_qc/v4/resoures/basics.py)
 vds = hl.vds.read_vds("gs://gnomad/raw/exomes/4.0/gnomad_v4.0.vds")
 meta_ht = hl.read_table("gs://broad-ukbb/broad.freeze_7/sample_qc/meta.ht")
 meta_ht = meta_ht.filter(meta_ht.sample_filters.high_quality)
 vds = hl.vds.filter_samples(vds, meta_ht, remove_dead_alleles=True)
 call_stats_ht = hl.read_table(ukb.var_annotations_ht_path('ukb_freq', *TRANCHE_DATA[CURRENT_TRANCHE]))
 freq_index = get_cohort_index(call_stats_ht)
 var = vds.variant_data.annotate_rows(call_stats=call_stats_ht[vds.variant_data.row_key].freq[freq_index])
-vds = hl.vds.variant_dataset.VariantDataset(vds.reference_data, var)
+# NOTE: If need to rerun, could use `from_merged_representation` method here
```

Comment:
the below line is fine using  hl.vds.VariantDataset. from_merged_representation converts from a sparse MatrixTable containing variant and reference data.